content/applications/general/digest_emails/create_digest_email.rst

```diff
@@ -0,0 +1,54 @@
+=============================
+Create your own digest emails
+=============================
+To create your own Digest Email, navigate to your **Settings** App and proceed to :menuselection:`General Settings --> Statistics --> Digest Settings`. Click **--> Configure Digest Emails** and then click **Create**.
+
+From there, give your Digest Email a title, specify periodicity, and choose your desired KPIs and recipients fields as needed. After you click **Save**, your new custom Digest Email will be available from the list under **Settings / Digest Emails** and in the dropdown menu on the General Settings screen.
+
+Add custom KPIs with Odoo Studio
+================================
```

Comment:
Hi again!

Additional note, now that I think of it! :)

For the link to documentation specific to custom KPIs, you can add an internal hyperlink target such as the suggestion here to have an anchor that won't change, even if you edit the title in RST later.

```suggestion
.. _digest-emails/custom-kpi::

Custom KPIs with Studio
=======================

```
the URL would be ----> ..../general/digest_emails.html#digest-email-custom-kpi. The devs could then use that link in the module itself rather than to the top of the page.